modules/nw-kube-proxy-config.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * networking/configuring-kubeproxy.adoc
+
+[id="modifying-kubeproxy-config-{context}"]
+= kube-proxy configuration parameters
+
+You can modify the following `kubeProxyConfig` parameters:
+
+.Parameters
+[cols=".^2,.^5,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`bindAddress`
+|The IP address for `kube-proxy` to serve on.
+|The default value is `0.0.0.0`.
```

Comment:
Are there any restrictions for this address or info about how a user's supposed to know the appropriate value?

Reply:
No longer exposing this parameter.